### PR TITLE
Snap2bytes

### PR DESCRIPTION
**Pull request by david-deboer**
This does a couple of things:
1 - primarily, convert the redis arrays to bytes rather than json strings, this also adds a diagnostic argument to jsonify
2 - it updates the hera-snap-redis-mon to be compatible with the new hera_corr_cm and not duplicate the redis info
3 - in blocks.py removes old, commented out roach modules
4 - in snap_fengine.py, it replaces the dangerous "from blocks import *" to give it a "snap_blocks" Namespace

### CODE REVIEW

File: control_software/src/snap_fengine.py [hunk 1]
```diff
@@ -17,9 +17,11 @@
 PAM_MIN = 0
 
 
-def _jsonify(val, cast=True):
+def _jsonify(var, val, cast=True, debug=False):
```
Comment (AaronParsons) [inline control_software/src/snap_fengine.py]: As far as I can see, adding var here is only for debug to print with. I don't think that usage warrants requiring all _jsonify calls to report a variable name, and passing the name as an argument strongly implies that it is used somehow in the jsonification process. Suggest reverting.

File: control_software/scripts/hera_snap_redis_monitor.py [hunk 2]
```diff
@@ -83,12 +54,15 @@ def print_ant_log_messages(corr):
     args = parser.parse_args()
 
     if args.loglevel not in ["DEBUG", "INFO", "WARNING", "ERROR"]:
-        logger.error("I don't undestand log level %s" % args.loglevel)
+        logger.error("I don't understand log level %s" % args.loglevel)
```
[the remark this answers is not in the document]
Comment (david-deboer) [reply control_software/scripts/hera_snap_redis_monitor.py:57]: I've changed to explict choices within the  arg parser.

File: control_software/scripts/hera_snap_redis_monitor.py [hunk 3]
```diff
@@ -133,67 +107,12 @@ def print_ant_log_messages(corr):
             {"version": __version__, "timestamp": datetime.now().isoformat()}
         )
 
-        if corr.r.exists("disable_monitoring"):
-            continue
         # Put full set into redis status:snap:<host>
         try:
-            corr.set_redis_status_fengs()
+            corr.set_redis_status_fengs(args.hosts)
```
[the remark this answers is not in the document]
Comment (david-deboer) [reply control_software/scripts/hera_snap_redis_monitor.py:112]: Removed

File: control_software/src/snap_fengine.py (hunk 1 above)
Comment (david-deboer) [reply control_software/src/snap_fengine.py]: I've reverted.

**Review by AaronParsons — CHANGES_REQUESTED** [review]
Something's not right with _cdjsonify, please check.

**Review by AaronParsons — APPROVED** [review]

**Pull request merged**